sdk/src/wallet/operations/block.rs

```diff
@@ -39,13 +41,23 @@ where
             }
         }
 
+        #[cfg(feature = "events")]
+        self.emit(WalletEvent::TransactionProgress(
+            TransactionProgressEvent::PreparedBlockSigningInput(unsigned_block.signing_input()),
```

Comment:
Don't want to prefix_hex::encode this? So it's also a String like `PreparedTransactionSigningHash`

Reply:
oh yes, I like the sound of that